microprofile/metrics-smallrye/src/main/resources/subsystem-templates/microprofile-metrics-smallrye.xml

```diff
@@ -3,8 +3,8 @@
 <!--  See src/resources/configuration/ReadMe.txt for how the configuration assembly works -->
 <config>
     <extension-module>org.wildfly.extension.microprofile.metrics-smallrye</extension-module>
-    <subsystem xmlns="urn:wildfly:microprofile-metrics-smallrye:1.0"
+    <subsystem xmlns="urn:wildfly:microprofile-metrics-smallrye:2.0"
                security-enabled="false"
-               exposed-subsystems="*">
-    </subsystem>
+               exposed-subsystems="*"
+               prefix="${wildfly.metrics.prefix:wildfly" />
```

Comment:
Should EAP have its own prefix or not ?

Reply:
it's likely but that's not a concern for WildFly.